libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss

```diff
@@ -1,64 +1,238 @@
+  --sky-override-text-editor-background-color: white;
+  --sky-override-text-editor-border-error: 1px solid
+    var(--sky-highlight-color-danger);
+  --sky-override-text-editor-border-focused: 1px solid
+    var(--sky-highlight-color-info);
+  --sky-override-text-editor-border-radius: 0;
+  --sky-override-text-editor-border: #{1px solid
+    $sky-border-color-neutral-medium};
+  --sky-override-text-editor-box-shadow-error: #{0 0 8px
+    rgba($sky-highlight-color-danger, 0.6)};
+  --sky-override-text-editor-box-shadow-focused: #{0 0 8px
+    rgba($sky-highlight-color-info, 0.6)};
+  --sky-override-text-editor-box-shadow: none;
+  --sky-override-text-editor-disabled-cursor: default;
+  --sky-override-text-editor-label-color: var(--sky-text-color-default);
+  --sky-override-text-editor-label-display: flex;
+  --sky-override-text-editor-label-font-size: 15px;
+  --sky-override-text-editor-label-font-weight: 400;
+  --sky-override-text-editor-label-line-height: normal;
+  --sky-override-text-editor-label-margin-bottom: 5px;
+  --sky-override-text-editor-label-padding: 0;
+  --sky-override-text-editor-toolbar-border: #{1px solid
+    $sky-border-color-neutral-medium};
+  --sky-override-text-editor-toolbar-item-spacing: 0 20px 10px 0;
+  --sky-override-text-editor-toolbar-section-padding: 10px 10px 0 10px;
+  --sky-override-text-editor-top-toolbar-bottom-padding: 0;
+  --sky-override-text-editor-wrapper-background-color-disabled: var(
+    --sky-background-color-disabled
+  );
+  --sky-override-text-editor-wrapper-padding: 1rem;
+}
+
+@include compatMixins.sky-default-overrides('.sky-text-editor-hint-text') {
+  --sky-override-text-editor-hint-text-space: 5px;
+}
+
+@include compatMixins.sky-modern-overrides('.sky-text-editor', false) {
+  --sky-override-text-editor-background-color: var(--modern-color-white);
+  --sky-override-text-editor-border-color: #{$sky-theme-modern-border-color-neutral-medium};
+  --sky-override-text-editor-focus-shadow: #{$sky-theme-modern-elevation-1-shadow-size
+    $sky-theme-modern-elevation-1-shadow-color};
+  --sky-override-text-editor-label-padding: 10px 10px 0 10px;
+  --sky-override-text-editor-toolbar-section-padding: #{$sky-theme-modern-space-sm};
+  --sky-override-text-editor-top-toolbar-bottom-padding: 0;
+}
+
+sky-text-editor.sky-form-field-stacked {
   display: block;
 }
 
 .sky-text-editor {
+  border-radius: var(
+    --sky-override-text-editor-border-radius,
+    var(--sky-border-radius-s)
+  );
+
+  box-shadow: var(
+    --sky-override-text-editor-box-shadow,
+    inset 0 0 0 var(--sky-border-width-input-base)
+      var(
+        --sky-override-text-editor-border-color,
+        var(--sky-color-border-input-base)
+      )
+  );
+
   &.sky-text-editor-disabled {
-    .sky-toolbar *,
-    .sky-text-editor-wrapper * {
-      color: var(--sky-text-color-deemphasized) !important;
-    }
+    background-color: var(--sky-color-background-input-disabled);
```

Comment:
Looks like the spec calls for tokens on the border here that I don't see. I don't see a functional difference when I tried it - but probably should still have the token.